src/builds.ts

```diff
@@ -98,39 +99,80 @@ export interface ProjectBuildOptions {
     /** Use babel to compile all ES6 JS down to ES5 for older browsers. */
     compile?: boolean
   };
+
+  /** Options relating to the browser that consumes this build. */
+  client?: {
+    /**
+     * Capabilities required for serving this build. Used by servers that
+     * support differential serving to decide which build to serve to a given
+     * user agent.
+     *
+     * Values include `es2015` and `push`. See canonical list at:
+     * https://github.com/Polymer/prpl-server-node/blob/master/src/capabilities.ts
+     */
+    capabilities?: string[];
+  };
+
+  /**
+   * Options for transforming resource references to facilitate serving this
+   * build from a non-root path. Useful for differential serving, where static
+   * resources for each build are served from sub-directories.
+   */
+  rewritePaths?: {
+    /**
+     * Use this prefix for rewriting. Defaults to the build's `name`.
+     * Leading/trailing slashes are optional.
+     */
+    prefix?: string,
+
+    /** Update the entrypoint's `<base>` tag, if found. */
+    baseTag?: boolean,
+
+    /** Prefix Service Worker pre-cached resources. */
+    serviceWorker?: boolean,
+
+    /* Prefix Push Manifest resources. */
+    pushManifest?: boolean
```

Comment:
In the previous commit, I just had a single "base" boolean|string that did all 3 transforms, but that seemed a bit blunt. Maybe it's not though -- just needs to be named something better?

In this commit, I was planning default off, in the interest of being explicit.